server/sonar-db-migration/src/test/java/org/sonar/server/platform/db/migration/version/v64/PopulateOrganizationMembersTableTest.java

```diff
@@ -0,0 +1,308 @@
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+package org.sonar.server.platform.db.migration.version.v64;
+
+import com.google.common.collect.ImmutableMap;
+import java.sql.SQLException;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import javax.annotation.Nullable;
+import org.apache.commons.lang.RandomStringUtils;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.sonar.core.util.stream.Collectors;
+import org.sonar.db.CoreDbTester;
+import org.sonar.server.platform.db.migration.version.v63.DefaultOrganizationUuidImpl;
+
+import static java.lang.String.format;
+import static org.apache.commons.lang.math.RandomUtils.nextLong;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class PopulateOrganizationMembersTableTest {
+
+  private static final String TABLE = "organization_members";
+
+  private static final String DEFAULT_ORGANIZATION_UUID = "def org uuid";
+
+  private static final String PERMISSION_PROVISIONING = "provisioning";
+  private static final String PERMISSION_ADMIN = "admin";
+  private static final String PERMISSION_BROWSE = "user";
+  private static final String PERMISSION_CODEVIEWER = "codeviewer";
+
+  private static final String ORG1_UUID = "ORG1_UUID";
+  private static final String ORG2_UUID = "ORG2_UUID";
+
+  private static final String USER1_LOGIN = "USER1";
+  private static final String USER2_LOGIN = "USER2";
+  private static final String USER3_LOGIN = "USER3";
+
+  @Rule
+  public ExpectedException expectedException = ExpectedException.none();
+
+  @Rule
+  public CoreDbTester db = CoreDbTester.createForSchema(PopulateOrganizationMembersTableTest.class, "initial.sql");
+
+  private PopulateOrganizationMembersTable underTest = new PopulateOrganizationMembersTable(db.database(), new DefaultOrganizationUuidImpl());
+
+  @Test
+  public void fails_with_ISE_when_no_default_organization_is_set() throws SQLException {
+    expectedException.expect(IllegalStateException.class);
+    expectedException.expectMessage("Default organization uuid is missing");
+
+    underTest.execute();
+  }
+
+  @Test
+  public void fails_with_ISE_when_default_organization_does_not_exist_in_table_ORGANIZATIONS() throws SQLException {
+    insertDefaultOrganizationUuid("blabla");
+
+    expectedException.expect(IllegalStateException.class);
+    expectedException.expectMessage("Default organization with uuid 'blabla' does not exist in table ORGANIZATIONS");
+
+    underTest.execute();
+  }
+
+  @Test
+  public void execute_has_no_effect_when_table_is_empty() throws SQLException {
+    setupDefaultOrganization();
+    underTest.execute();
```

Comment:
suggestion: as much as possible, organize your code following the [Arrange Act Assert](http://wiki.c2.com/?ArrangeActAssert) pattern which is convenient once we agreed on the schema.
The drawback is it's not applicable with the ExpectedException where it would be Arrange Assert Act
I then generally separate by one line each part